### ISSUE TITLE
Cannot read property 'INFO' of undefined

### ISSUE THREAD

**Issue by jlengstorf** [log4js-node/log4js-node#652]
Unless I'm missing something, there was a breaking change between 2.3.7 and 2.5.1.

We have a logger utility that starts out with this code:

```js
const log4js = require('log4js');
// ... other requires ...

const levels = log4js.levels;
let level = levels.INFO;
```

In 2.3.7, this runs as expected. However, if I upgrade to log4js@2.5.1, it throws a `TypeError`:

```bash
TypeError: Cannot read property 'INFO' of undefined
```

This is thrown on the `let level = levels.INFO` line.

Did something change and I missed the memo? Or is this a bug?

Thanks!

**Comment by nomiddlename** [comment]
I think this is a problem introduced by #648. Levels is set up by `configure`, which used to get called when log4js was required. Now it doesn't get called unless you call it explicitly or on the first call to `getLogger`.

As a workaround, would you be able to call `log4js.getLogger()` before getting a reference to `log4js.levels`?

**Comment by jlengstorf** [comment]
That workaround does clear up the error, yeah. That does seem like it should be considered a breaking change, though, yeah?

**Comment by nomiddlename** [comment]
It's a bug, the change was not intended to change the way the API worked but we missed the exports defined during configure.

**Comment by jlengstorf** [comment]
Ah, gotcha. Thanks for the quick replies!

**Comment by nomiddlename** [comment]
Published a fix for this to NPM v2.5.2

**Comment by jlengstorf** [comment]
Confirmed fixed in several build pipelines. Thanks for the quick patch!
